content/javascript/concepts/objects/terms/isExtensible/isExtensible.md

```diff
@@ -0,0 +1,75 @@
+---
+Title: '.isExtensible()'
+Description: 'Returns a boolean value depending on whether a given object can have properties added to it.'
+Subjects:
+  - 'Web Development'
+  - 'Code Foundations'
+
+Tags:
+  - 'Methods'
+  - 'Objects'
+CatalogContent:
+  - 'introduction-to-javascript'
+  - 'paths/full-stack-engineer-career-path'
+---
+
+The **`.isExtensible()`** is a static method that returns `true` if the given object can have properties added to it and `false` if the object cannot have properties added to it.
+
+## Syntax
+
+```pseudo
+Object.isExtensible(newObject);
+```
+
+Objects are extensible by default. An object can be made non-extensible by using `Object.freeze()`, `Object.preventExtensions`, ot `Object.seal()`. A non-extensible object can still have its properties modified.
```

Comment:
Add this as a note, take reference on how to add a note from Codecademy's [style guide](https://github.com/Codecademy/docs/blob/main/documentation/style-guide.md).